clients/lsp-beancount.el

```diff
@@ -0,0 +1,68 @@
+;;; lsp-beancount.el --- Beancount Client settings         -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2021 emacs-lsp maintainers
+
+;; Author: emacs-lsp maintainers
+;; Keywords: lsp, beancount
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP client for Beancount
+
+;;; Code:
+
+(require 'lsp-mode)
+
+(defgroup lsp-beancount nil
+  "Settings for the Beancount Language Server."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/polarmutex/beancount-language-server")
+  :package-version '(lsp-mode . "7.1.0"))
+
+(defcustom lsp-beancount-langserver-executable "beancount-langserver"
+  "Command to start Beancount language server."
+  :type 'string
+  :group 'lsp-beancount
+  :package-version '(lsp-mode . "7.1.0"))
+
+(defcustom lsp-beancount-journal-file nil
+  "Pathg to Beancount journal file."
+  :type 'string
+  :group 'lsp-beancount
+  :package-version '(lsp-mode . "7.1.0"))
+
+(defcustom lsp-beancount-python-interpreter
```

Comment:
can you leave that `nil` and then do the executable-find in `lambda` in lsp-stdio-connection? This will avoid calling executable-find which will scan the classpath when loading the packages.

Reply:
Sure. Make sense.